CMake/cdat_modules_extra/setup_runtime.sh.in

```diff
@@ -1,6 +1,18 @@
 # Everything beyond this point will be determined relatively
 # from this path.
 install_prefix="@CMAKE_INSTALL_PREFIX@"
+# Reset previous uvcdat env messing up
+source @CMAKE_INSTALL_PREFIX@/bin/reset_runtime.sh
+
+# Saves what we will mess with
+for v in PATH LD_LIBRARY_PATH DYLD_LIBRARY_PATH PYTHONPATH OPAL_PREFIX LIBOVERLAY_SCROLLBAR ; do
+   tmp="${v}"
+    if [ -n "${!tmp}" ] ; then
```

Comment:
I'd say, we can merge this and say that these scripts work only for (ba)sh and (t)csh shells and create a new issue to evaluate if we really need source scripts for other shells (based on their popularity among UV-CDAT users/developers). What do you think?